demos/asp-net-core/README.md

```diff
@@ -2,7 +2,7 @@
 
 This demo is composed of two solutions which demonstrate UI Composition techniques using ASP.NET Core.
 
-In order to run the demos, you require the latest .NET Core Runtime in the [1.1.x version range](https://www.microsoft.com/net/download/all).
+In order to run the demos, you require the latest .NET Core Runtime in the [2.x version range](https://www.microsoft.com/net/download/all).
```

Comment:
I'm not sure that `netcoreapp2.0` projects will run against the 2.1 runtime. This may have to be the 2.0.x version range.

Reply:
are you saying that runtime are not backward compatible? :trollface:

Reply:
fixed